locales/es.po

```diff
@@ -48,73 +48,73 @@ msgstr "Intentaste anidar más expresiones de las que estan permitidas actualmen
 
 #: src/callbacks.py:773
 msgid "The command %q is available in the %L plugins.  Please specify the plugin whose command you wish to call by using its name as a command before %q."
-msgstr "El comando %q está disponible en %L. Por favor especifica"
+msgstr "El comando %q está disponible en los complementos de %L. Por favor especifica el comando que quieres usar de este complemento usando su nombre como comando antes de %q"
 
 #: src/callbacks.py:869 src/callbacks.py:882
 msgid "(XX more messages)"
-msgstr "FALTA TRADUCCION"
+msgstr "(XX mensajes más)"
 
 #: src/callbacks.py:914
 msgid "more message"
-msgstr "FALTA TRADUCCION"
+msgstr "mensaje más"
 
 #: src/callbacks.py:916
 msgid "more messages"
-msgstr "FALTA TRADUCCION"
+msgstr "mensajes más"
```

Comment:
les confirmo la traduccion es literal ya que la linea de codigo dice:

`if not allowedLength: # 0 indicates this.
                        allowedLength = 470 - len(self.irc.prefix)
                        allowedLength -= len(msg.nick)
                        # The '(XX more messages)' may have not the same
                        # length in the current locale
                        allowedLength -= len(_('(XX more messages)'))
`
literalmente hablamos de una cantidad de mensajes más.